README.md

```diff
@@ -30,12 +30,14 @@ Building
 
 You need to install both OpenResty and [PostgreSQL](http://www.postgresql.org/) 9.x, obviously.
 
-You need to install [pandoc](http://pandoc.org/) and [marked](https://marked.js.org/) to your system.
+You need to install [pandoc](http://pandoc.org/), [marked](https://marked.js.org/), [cheerio](http://www.npmjs.com/package/cheerio) and [axios](https://www.axios.com/) to your system.
 On Mac OS X, for example, it's as simple as
 
 ```bash
 brew install pandoc
 npm install marked
+npm install cheerio
+npm install axios
```

Comment:
Can we combine these 3 `npm install` commands into a single one? It'll be faster and easier.

Reply:
combined.